src/devices/Common/Iot/Device/Multiplexing/VirtualOutputSegment.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.Gpio;
+using System.Threading;
+
+namespace Iot.Device.Multiplexing.Utility
+{
+    /// <summary>
+    /// Interface that abstracts multiplexing over a segment of outputs.
+    /// </summary>
+    public class VirtualOutputSegment : IOutputSegment
```

Comment:
First: Description is missleading, this is not an interface. 
Second: It seems this should rather be named `NullOutputSegment` as it does nothing except discarding all inputs.
Third: Seal classes not designed for inheritance.

Reply:
Will make it sealed. `Virtual` seems better than `Null`. It is the backing store for all the implementations I wrote. Is there a better name for that?

Reply:
If it's a backing store, `Virtual` is probably ok, or something like `MemoryMapped`. However, what confused me more is that seemingly nothing is ever read back from this class (initially, it didn't even have a way to get values back), so maybe you implemented it differently than what your intent was.

Reply:
Ok, found it. It's hidden a bit in that Write() loop, but got it now.